internal/pkg/service/buffer/api/service/mapper/mapper.go

```diff
@@ -81,7 +81,7 @@ func (m Mapper) MappingPayloadFromModel(model model.Mapping) buffer.Mapping {
 	}
 }
 
-func (m Mapper) ReceiverModelFromPayload(projectID int, secret string, payload buffer.CreateReceiverPayload) (r model.Receiver, err error) {
+func (m Mapper) ReceiverModelFromPayload(projectID int, sapiToken string, secret string, payload buffer.CreateReceiverPayload) (r model.Receiver, err error) {
```

Comment:
Skratka `sapi` je "zakazana"v Keboola :slightly_smiling_face: , daj prosim iba `token`, alebo `storageAPIToken` (ako to je v celom projekte).